src/components/Input/components/Switch/index.tsx

```diff
@@ -0,0 +1,25 @@
+import React, { ReactElement } from 'react';
+import clsx from 'clsx';
+import InputBase from '../InputBase';
+import { Props } from './types';
+
+
+function Switch({ className, ...otherProps }: Props): ReactElement {
+    const switchClasses = clsx(
+        'brew-Input--Switch',
+        className,
+    );
+    const customControl = (
```

Comment:
Since these two constants are unrelated to each other, I would advise adding padding between them for readability purposes.